spec/requests/cash_payment_forms_spec.rb

```diff
@@ -83,7 +83,7 @@
 
         transient_registration.reload
 
-        expect(response).to redirect_to(resource_finance_details_path(transient_registration._id))
+        expect(response).to redirect_to(resource_finance_details_path(transient_registration.registration._id))
```

Comment:
Total nitpick, but just wondering if there's a reason why we do this test differently to the other payment ones, where we assign `registration = transient_registration.registration` and then use that in the route?

Reply:
In the other tests, we do that because calling the `let` variable after the object has been deleted (so after the request)  causes the object to be recreated and inconsistent test results. Is a bug we have discussed some months ago. In this case, we are stubbing out the renew job to actually never run, hence we never delete the transient_registration, hence I don't need to ensure that I never call the let variable after the object have been deleted by the request. Is one of those "I have to put a dot at the refactoring somewhere" so this will need more attention later. Changing this test to follow the common pattern means re-writing a lot of it.

Reply:
NM that, after all I have made the refactoring!